lib/refile/attachment/active_record.rb

```diff
@@ -24,17 +24,17 @@ def attachment(name, raise_errors: false, destroy: true, **options)
         end
 
         define_method "#{name}=" do |value|
-          send("#{name}_id_will_change!") if respond_to?("#{name}_id_will_change!")
+          send("will_save_change_to_#{name}_id?") if respond_to?("#{name}_id_will_change!")
```

Comment:
I don't believe this is equivalent.

the bang method changes the value
the ? only returns a boolean

```
(byebug) will_save_change_to_review_category_cache?
false
(byebug) review_category_cache_will_change!
DEPRECATION WARNING: The behavior of `attribute_changed?` inside of after callbacks will be changing in the next version of Rails. The new return value will reflect the behavior of calling the method after `save` returned (e.g. the opposite of what it returns now). To maintain the current behavior, use `saved_change_to_attribute?` instead. (called from refresh_state! at (byebug):1)
#<Set: {"review_category_cache"}>
(byebug) will_save_change_to_review_category_cache?
true
(byebug) 
```